Makefile

```diff
@@ -113,6 +113,20 @@ ifeq ($(ENABLE_FRAMEBUFFER), 1)
 	NASM_OPTIONS += -dENABLE_FRAMEBUFFER
 endif
 
+ifeq ($(ENABLE_FRAMEBUFFER_STATS), 1)
+	CFLAGS += -DENABLE_FRAMEBUFFER_STATS
+endif
+
+ifeq ($(ENABLE_FRAMEBUFFER_DEMO), 1)
```

Comment:
I'd suggest we add the demo in a different patch.

Reply:
Will remove

Reply:
Done!